### ISSUE TITLE
Android API 33 : Original error: 'de.rm.app.android.debug' is still running after 500ms timeout

### ISSUE THREAD

**Issue by StephanB75** [appium/appium#17444]
## The problem

I have under Android API 33 sometimes the issue that on driver.terminateApp() throws an error.
Original error: 'de.rm.app.android.debug' is still running after 500ms timeout

It does not occur everytime.
is it possible to make the timeout configurable

## Environment

* Appium version (or git revision) that exhibits the issue: 2.0 beta43
* Last Appium version that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Appium: Monterey
* Node.js version (unless using Appium.app|exe): 16.16
* Npm or Yarn package manager: 8.11
* Mobile platform/version under test: Android
* Real device or emulator/simulator: Emulator
* Appium CLI or Appium.app|exe: CLI

## Link to Appium logs

(https://gist.github.com/StephanB75/1dfbe28ada1c99f7f0dc4e5e1d5e7d5d)

**Comment by mykola-mokhnach** [comment]
Yes, the server API does provide such possibility: https://github.com/appium/appium-android-driver/blob/d6b1b4c8cc30af66b3389dff0bf4b210b25561fd/lib/commands/app-management.js#L130

**Comment by StephanB75** [comment]
@mykola-mokhnach can you give me a hint on the java syntax ?

**Comment by KazuCocoa** [comment]
https://github.com/appium/java-client/blob/bd66144e8a15ec9bbe1af8a4c25087d527b79e90/src/main/java/io/appium/java_client/InteractsWithApps.java#L101-L116 ?

**Comment by StephanB75** [comment]
@KazuCocoa Thx but i still don't get it how to define a BaseTerminateApplicationOptions to append it to the terminateApp command :(
